infrastructure/cdn-in-a-box/traffic_ops_integration_test/Dockerfile

```diff
@@ -46,7 +46,7 @@ WORKDIR /go/src/github.com/apache/trafficcontrol/traffic_ops/testing/api
 RUN go mod vendor -v
 
 RUN set -o errexit -o xtrace;\
-    for api_version in v3 v4; do\
+    for api_version in v3 v4 v5; do\
```

Comment:
`v3 v4 v5` can be `v{3..5}` like https://github.com/apache/trafficcontrol/blob/e9616117a52277f6d28bfee12873496d68e1ecae/infrastructure/cdn-in-a-box/traffic_ops_integration_test/run.sh#L45